timeline.tex

```diff
@@ -1,39 +1,22 @@
-It will continue to be updated as Rubin Construction and the Early Science Program progress. 
+Table~\ref{tab:ops-timeline} shows the Operations timeline and provides nominal date ranges for the various elements of the Early Science Program.
 The date ranges are derived from the Rubin ``Celebratory Milestones'', which are  published monthly on the Rubin Project website\footnote{\url{ls.st/dates}}. 
-
-\begin{table}[ht]
-\centering
-\includegraphics[width=\linewidth]{figures/DPR-milestones}
-\caption{Top milestones for the Early Science Program.}
-\label{tab:milestones}
-\end{table}
-
-Milestone dates are given as min-max ranges to indicate the associated uncertainty. 
+For completed milestones the delivered date is given. 
+Milestone dates are given as min -- max ranges to indicate the associated uncertainty. 
 Typically the near date corresponds to the current Project forecast, plus any additional operational uncertainty.
-The late date corresponds (approximately) to the current Project ``late date'' plus any additional operational uncertainty.
-An intermediate (typically mid-range) date is used by the Rubin Operations teams for planning purposes. 
-
-The LSST survey start is currently expected to be sometime between February 2025 and September 2025.
-The timing of the Commissioning observations is somewhat less uncertain and the timing of the release of those data to the community can be projected to within a few months at the time of writing.
-
-Table \ref{tab:timeline} shows the nominal date ranges for the various elements of the Early Science Program. 
+The late date corresponds approximately to the current Project late date plus any additional operational uncertainty.
 The shaded region contains roughly 80\% of the probability, while the lefthand edge of the shaded range indicates the earliest date the milestone could be reached. 
-The darker regions give a very rough indication of the +/-1 sigma error bars. 
-Over the course of the commissioning period, we expect these shaded regions to shrink as our understanding of the remaining schedule uncertainty improves. 
-However, there is still the possibility of the assumptions underlying these distributions being wrong: this is just our best estimate at the current time.
+The darker regions give a very rough indication of the +/-1 sigma error bars.
+Over the course of the commissioning period, we expect these data ranges to shrink as our understanding of the remaining schedule uncertainty improves. 
+However, there is still the possibility of the assumptions underlying these distributions being wrong; this is just our best estimate at the current time.
+
+\input{tables/rubin_ops_timeline}
 
-The next key milestone in the Early Science Program is the release of DP0.3. 
-Scheduled for mid-2023, DP0.3 will be the last in the DP0 series, which is based on simulated LSST-like data. 
+The next key milestone in the Early Science Program is the release of DP1.
```

Comment:
It's odd that the next sentence gives a date window, for something further in the future, but this one does not.